lib/engines/content_block_manager/app/models/content_block_manager/content_block/edition/has_audit_trail.rb

```diff
@@ -28,8 +28,73 @@ def record_update
       unless draft?
         user = Current.user
         state = try(:state)
-        versions.create!(event: "updated", user:, state:)
+        versions.create!(event: "updated", user:, state:, changed_fields:)
       end
     end
+
+    def changed_fields
+      if document.editions.count <= 1
+        nil
+      else
+        @previous_edition = document.editions[-2]
+        diff = []
+        diff.append(diff_for_lead_organisation)
```

Comment:
organisation proved to be a bit complicated as I wasn't able to call `lead_organisation` on the previous edition directly...

Reply:
You might be able to get `lead_organisation` if you add an `includes` to the call to `editions`. Something like:

```
editions.includes(:edition_organisation, :organisation)
```